Automattic/wp-calypso
# Upgrade ESlint plugin inclusive language and allow some terms

#### Changes proposed in this Pull Request

* Upgrade `inclusive-language` to get the new `allowedTerms` feature
* Add `masterbar` and `mastercard` as allowed terms.

#### Testing instructions

* Open up a file that contains one of the allowed terms and ensure that there are no ESLint warnings.
